k-distribution/tests/regression-new/kast-rule/kast-in2.kast

```diff
@@ -0,0 +1 @@
+<k> init(0) => 2 </k>
```

Comment:
This looks like it is evaluating the `init()` function. ... or is that a macro now?